src/stories/helpers/add-notes.ts

```diff
@@ -1,5 +1,5 @@
-export default (notes) => {
+export default function addNotes(notes) {
     return {
         notes,
     };
```

Comment:
my tslint config says it’s missing a semicolon.

Reply:
my tslint config says it’s an unnecessary semicolon :P

Reply:
personally i prefer without semicolons. it seems that the tslint config is conflicting with different settings ... something else to sort out later.